aspnetcore/razor-pages/index.md

```diff
@@ -98,6 +98,84 @@ The *MyApp/Pages/Contact.cshtml* file:
 ```html
 @page
 @using MyApp
+@using MyApp.Pages
```

Comment:
This might be an OK stopgap measure, but ultimately we want to import all snippets from a working sample. We generally don't allow copy/paste of code. I can work on that sample. Thanks for your contribution.